app/views/initiatives/_initiative.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="card">
+  <h3 class="init-name"><%= initiative.title %></h3>
+  <p class="init-description no-link"><%= simple_format(initiative.description) %></p>
+  <div>
+    <div class="init-footer w-row">
+      <div class="w-col w-col-2 padding-zero"><%= image_tag "icon-#{initiative.completion ? 'completion': 'calendar'}.svg", class: "icon-target" %></div>
+      <div class="card-date w-col w-col-10">
+        <%= initiative.target_date.to_s(:long_ordinal) unless initiative.target_date.nil? %>
+      </div>
+      <div class="w-col w-col-2 padding-zero">
+        <% if !current_user.team_members.pluck(:id).include?(params[:user_id].to_i) %>
```

Comment:
How can we move this logic out of the view?

Reply:
change the logic